parcel.go

```diff
@@ -35,26 +60,64 @@ func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
-
+	rows, err := s.db.Query("SELECT number, client, status, address, created_at FROM parcel WHERE client = :client", sql.Named("client", client))
+	if err != nil {
+		return res, err
```

Comment:
Лучше возвращать пустые значения, когда ошибка
```suggestion
		return nil, err
```